apps/web/src/pages/ApplicantDashboardPage/components/ReviewApplicationPreviewList.tsx

```diff
@@ -0,0 +1,578 @@
+import { useIntl } from "react-intl";
+import { differenceInDays } from "date-fns/differenceInDays";
+import { useRef, useState } from "react";
+
+import {
+  graphql,
+  PoolSkillType,
+  ReviewApplicationPreviewListFragment,
+} from "@gc-digital-talent/graphql";
+import {
+  commonMessages,
+  getLocale,
+  getLocalizedName,
+} from "@gc-digital-talent/i18n";
+import {
+  Accordion,
+  Button,
+  Dialog,
+  Link,
+  PreviewList,
+  PreviewMetaData,
+  Separator,
+  Well,
+} from "@gc-digital-talent/ui";
+import { formatDate, parseDateTimeUtc } from "@gc-digital-talent/date-helpers";
+import { unpackMaybes } from "@gc-digital-talent/helpers";
+
+import FieldDisplay from "~/components/ToggleForm/FieldDisplay";
+import talentRequestMessages from "~/messages/talentRequestMessages";
+import processMessages from "~/messages/processMessages";
+import { getClassificationName } from "~/utils/poolUtils";
+import {
+  ApplicationStatus,
+  getApplicationStatusChip,
+  getSalaryRange,
+} from "~/utils/poolCandidate";
+import { sortPoolSkillsBySkillCategory } from "~/utils/skillUtils";
+import useRoutes from "~/hooks/useRoutes";
+
+import { ApplicationDate } from "./MetadataDate";
+import StatusSummary from "./StatusSummary";
+
+export const ReviewApplicationPreviewList_Fragment = graphql(/* GraphQL */ `
+  fragment ReviewApplicationPreviewList on PoolCandidate {
+    id
+    expiryDate
+    finalDecisionAt
+    submittedAt
+    removedAt
+    status {
+      value
+      label {
+        en
+        fr
+      }
```

Comment:
If you want, you can replace all of these `{ en fr }` with just `{ localized }`

Reply:
Changed in [d3df6a8](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12366/commits/d3df6a87e7ff109b65f0f9d03590c960f09407b5)

Reply:
I see localzied was added but `en` and `fr` were not removed. If we need those then it isn't much benefit to do `localized` :sweat_smile: 

If not, then they should be removed so we don;t over fetch

Reply:
I thought they were required :sweat_smile: Fixed in [eabc236](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12366/commits/eabc2365100916752a224bf0c454f02bad65d8ea)